client/post-editor/controller.js

```diff
@@ -115,6 +115,7 @@ module.exports = {
 
 		function startEditing( siteId ) {
 			context.store.dispatch( setEditorPostId( postID ) );
```

Comment:
I think what you proposed: `getNewEditorPath` selector, is a good approach to solve this definitely.

Anyway, Should we merge the current approach of this branch?